doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,414 @@
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method = list
+      set list x           = 0
+      set list y           = 0.08
+      set list z           = 0
+    end
+
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-particles`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-particles initial-particle.prm
+
+.. image:: images/particle_insertion.png
+    :alt: inserted particle at the top of the channel
+    :align: center
+
+After the particle has been inserted it is now possible to simulate its sedimentation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is to be carried out using the particle inserted within the previous step. We will discuss the different parameter file sections. We introduce the different sections of the parameter file ``single-particle-sedimentation.prm`` needed to run this simulation. Most subsections are explained in detail in other CFD-DEM examples such as:  `Gas-solid spouted bed <../../../examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for :math:`2` s with a time step of :math:`0.005` s. The time scheme chosen for the simulation is first order backward difference method (BDF1). The simulation control section is shown:
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set number mesh adapt    = 0
+      set output name          = result_
+      set output frequency     = 20
+      set time end             = 2
+      set time step            = 0.005
+      set output path          = ./output/
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties subsection allows us to determine the density and viscosity of the fluid. We choose a density of :math:`996.8` kg/m\ :sup:`3` and a kinematic viscosity of :math:`0.0000008379` m\ :sup:`2`/s as to simulate the particle sedimentation in water.
+
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.0000008379
+        set density             = 996.8
+      end
+    end
+
+Initial Conditions
+~~~~~~~~~~~~~~~~~~
+
+We choose zero initial conditions for the velocity.
+
+.. code-block:: text
+
+    subsection initial conditions
+      set type = nodal
+      subsection uvwp
+        set Function expression = 0; 0; 0; 0
+      end
+    end
```

Comment:
```suggestion
Initial Conditions
~~~~~~~~~~~~~~~~~~

We choose zero initial conditions for the velocity, which is the default initial condition.

.. code-block:: text
```
This is the default initial condition, no need to specify it. I would literally remove that whole part.